_layouts/default.html

```diff
@@ -1,6 +1,28 @@
----
-theme :
-  name : bootstrap
----
-{% include JB/setup %}
-{% include themes/bootstrap/default.html %}
+<!doctype html>
+<html lang="{{ site.lang | default: "en-US" }}">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+
+{% seo %}
+    <link rel="stylesheet" href="{{ '/assets/css/style.css?v=' | append: site.github.build_revision | relative_url }}">
+    <script src="https://code.jquery.com/jquery-1.12.4.min.js" integrity="sha256-ZosEbRLbNQzLpnKIkEdrPv7lOy9C27hHQ+Xp8a4MxAQ=" crossorigin="anonymous"></script>
```

Comment:
Why is jquery required?

Reply:
I should have explained this a bit - default.html is actually what comes with the theme with some modifications (based on [this](https://github.com/pages-themes/midnight/blob/master/README.md#layouts) recommendation. The theme included default template has a link to the github repo right on top and downloading it  etc. See this [sample](https://pages-themes.github.io/midnight/)
The changes I made is to override that default template to remove that top section.